doc/source/user/basics.types.rst

```diff
@@ -96,70 +96,17 @@ The primitive types supported are tied closely to those in C:
 
 
 Since many of these have platform-dependent definitions, a set of fixed-size
-aliases are provided:
+aliases are provided to this site: :doc:`../reference/arrays.scalars`
+under the "Sized aliases" section.
```

Comment:
I suggest adding a proper link to this section. This means the current text will read 
```suggestion
aliases are provided (See :ref:`sized-aliases`).
```

You will also need to edit the file `doc/source/reference/arrays.scalars.rst` and just before the "Sized aliases" section you should add the following:

`.. _sized-aliases:`

FInally, you should always build the documentation locally to check that these references and links are working. If you have trouble doing that please mention it here and I can help.

Reply:
Fixed.